aspnetcore/host-and-deploy/web-farm.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Host ASP.NET Core in a web farm
+author: guardrex
+description: Learn how to host multiple instances of an ASP.NET Core app with shared resources in a web farm environment.
+ms.author: riande
+ms.custom: mvc
+ms.date: 07/16/2018
+uid: host-and-deploy/web-farm
+---
+# Host ASP.NET Core in a web farm
+
+By [Luke Latham](https://github.com/guardrex) and [Chris Ross](https://github.com/Tratcher)
+
+A *web farm* is a group of two or more web servers (or *nodes*) that host multiple instances of an app. When requests from users arrive to a web farm, a *load balancer* distributes the requests to the web farm's nodes. Web farms improve:
+
+* Reliability/availability &ndash; When one or more nodes fail, the load balancer can route requests to other functioning nodes to continue processing requests.
```

Comment:
Consider bolding the text before each ndash. This will make it a bit more prominent and easier to pick out if scanning the document.